RevenueCatUI/CustomerCenter/Views/PurchaseCardView.swift

```diff
@@ -153,14 +159,22 @@ struct PurchaseInformationCardView: View {
                               ? UIColor.systemBackground
                               : UIColor.secondarySystemBackground))
 
-            if let additionalInfo {
-                Text(additionalInfo)
-                    .font(.caption)
-                    .foregroundStyle(.primary)
-                    .padding(.bottom, 4)
-                    .frame(maxWidth: .infinity, alignment: .leading)
-                    .multilineTextAlignment(.leading)
-                    .padding()
+            if let additionalInfo, let additionalIcon {
+                HStack(alignment: .center, spacing: 12) {
```

Comment:
Question: should this be a `CompatibilityLabeledContent`? I guess it's not a big deal

Reply:
It's the opposite. The icon is on the left.